docs/install.md

```diff
@@ -106,3 +107,34 @@ That way you have control over when a breaking change affects you.
     # │        │        │        │        │        │        │        │        │        │        │
     # (1, 0)───(1, 1)───(1, 2)───(1, 3)───(1, 4)───(1, 5)───(1, 6)───(1, 7)───(1, 8)───(1, 9)───(1, 10)
     ```
+
+
+### Installing on Docker
```

Comment:
Because this creates a development environment, this should be moved into the developer documentation. https://github.com/quantumlib/Cirq/blob/master/docs/development.md

We may want a different dockerfile for just the user install.

Reply:
I think this Dockerfile should be the user install; which could be done by getting rid of the `git clone` RUN command and adding `cirq` to the `pip install` RUN command. As far as i know, none of us use Docker for development. If needed, we could have the appropriate Dockerfile in `dev_tools`.

Reply:
Yes, putting the dev dockerfile in dev_tools and having this one be a user install is a good idea.